### ISSUE TITLE
Check parquet output

### ISSUE THREAD

**Issue by domoritz** [domoritz/csv2parquet#2]
For example, the nation table seems to produce corrupt parquet.

**Comment by Dakror** [comment]
[nation.txt](https://github.com/domoritz/csv2parquet/files/6217860/nation.txt)

**Comment by domoritz** [comment]
Running `csv2parquet nation.txt nation.parquet -d '|' -h false`

**Comment by domoritz** [comment]
Hmm, even this simple CSV file is invalid

```
a,b,c
0,1,hello world
0,1,hello world
0,1,hello world
0,1,hello world
0,1,hello world
0,1,hello world
0,1,hello world
```

**Comment by domoritz** [comment]
Filed https://issues.apache.org/jira/browse/ARROW-12124

**Comment by nevi-me** [comment]
Hi @domoritz, you have to call `writer.close()` (https://docs.rs/parquet/3.0.0/parquet/arrow/arrow_writer/struct.ArrowWriter.html#method.close). This writes the metadata to the footer of the tile.

WIthout the footer, you can't read the file as you don't have the page offset information (and schemas, etc).

**Comment by domoritz** [comment]
Thank you. Fixed in 3821deb
